en-wordlist.txt

```diff
@@ -230,6 +233,7 @@ elementwise
 embeddings
 encodings
 ensembling
+enum
```

Comment:
Can you also add: 
rollouts
rollout
Ornstein
Uhlenbeck
OU

This should be enough to pass the spellcheck after merging other suggestions